packages/client/lib/util/parse.ts

```diff
@@ -114,7 +115,7 @@ async function createGethGenesisStateTrie(alloc: any) {
     const { balance, code, storage } = value as any
     const account = new Account()
     if (balance) {
-      account.balance = new BN(isHexPrefixed(balance) ? toBuffer(balance) : balance)
+      account.balance = toType(balance, TypeOutput.BigInt) ?? BigInt(0)
```

Comment:
toType, like toBuffer, doesn't support decimal strings, so I would do this, since if 0x-prefixed string it will interpret as base 16 and if not it will do base 10

```suggestion
      account.balance = BigInt(balance)
```